src/tests/test_integration/test_comments.py

```
@@ -0,0 +1,189 @@
+        "parent_id": parent_id
+    }, headers=headers2)
+    assert reply_resp.status_code == 200
+
+    notif_resp = await client.get("/api/v1/comments/notifications/", headers=headers1)
+    assert notif_resp.status_code == 200
+    notifications = notif_resp.json()
+    assert any("replied" in n["message"] for n in notifications)
+
+
+@pytest.mark.asyncio
+async def test_like_comment_creates_notification(
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)